src/main/java/appeng/container/implementations/ContainerQuartzKnife.java

```diff
@@ -156,15 +156,22 @@ public void putStack( final ItemStack stack )
 
 		private void makePlate()
 		{
-			if( !this.getItemHandler().extractItem( 0, 1, false ).isEmpty() )
+			if( Platform.isServer() )
 			{
-				final ItemStack item = ContainerQuartzKnife.this.toolInv.getItemStack();
-				item.damageItem( 1, ContainerQuartzKnife.this.getPlayerInv().player );
-
-				if( item.getCount() == 0 )
+				if( !this.getItemHandler().extractItem( 0, 1, false ).isEmpty() )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1066)